.rubocop.yml

```diff
@@ -2,33 +2,29 @@ require:
   - rubocop-packaging
 
 AllCops:
+  NewCops: enable
   Exclude:
     - bundle/vendors/**/*
+  TargetRubyVersion: 2.4
```

Comment:
I am not sure why the CI failed: https://travis-ci.org/github/arnau/ISO8601/jobs/705068359
Any clue?

Reply:
`Error: RuboCop found unsupported Ruby version 2.1 in `TargetRubyVersion` parameter (in vendor/bundle/ruby/2.7.0/gems/rainbow-3.0.0/.rubocop.yml). 2.1-compatible analysis was dropped after version 0.57.`
Even though the TargetRubyVersion is 2.4 :sweat_smile:

Reply:
I'm not sure either 🤔